src/twisted/words/xish/domish.py

```diff
@@ -13,8 +13,7 @@
 
 from zope.interface import implementer, Interface, Attribute
 
-from twisted.python.compat import (StringType, _coercedUnicode,
-                                   iteritems, itervalues)
+from twisted.python.compat import StringType, iteritems, itervalues
```

Comment:
StringType, iteritems, and itervalues can all be easily dropped.
You can do that in this PR, or do a separate one if you want.

Reply:
I want to handle the public definitions in a separate PR, because if I understand the compatibility policy correctly, those should all be deprecated instead of removed.